web-app/src/components/NationPreferencesDialog.tsx

```diff
@@ -91,9 +83,9 @@ const NationPreferencesDialog = (): React.ReactElement => {
 
   return (
     <Dialog open={open} onClose={close}>
-      <DialogTitle>{NATION_PREFERENCES_DIALOG_TITLE}</DialogTitle>
+      <DialogTitle>{t(tk.nationPreferences.title)}</DialogTitle>
       <DialogContent>
-        <Typography>{NATION_PREFERENCES_DIALOG_PROMPT}</Typography>
+        <Typography>{t(tk.nationPreferences.dialog)}</Typography>
```

Comment:
I think `dialog` is not a meaningful name for this variant. In an ideal world we want someone to be able to look at the transation file and understand pretty much straight away what each variable is. I would suggst `prompt` or `subtitle` but there are probably other good names.